templates/partials/nav.html

```diff
@@ -15,5 +15,8 @@
         {% for menu in config.extra.menu | default(value=[]) %}
         <a href={{ get_url(path=menu.url) }} style="margin-left: 0.5em">{{ menu.name }}</a>
         {% endfor %}
+
+        <a href="{{ config.base_url }}" style="margin-left: 1em">EN</a>
```

Comment:
These links still needs to be active/inactive depending on which language is currently selected.